qcengine/config.py

```diff
@@ -82,13 +110,16 @@ class Config:
 
 
 class TaskConfig(pydantic.BaseModel):
+    """Description of the configuration used to launch a task."""
 
     # Specifications
-    ncores: int  # Number of ncores per task
+    ncores: int  # Number of cores per node
```

Comment:
This is truly core per task. We often run where ncores < ncores per node.

Reply:
Ah, I'll change it back and add some notes that this means "number of cores per node" for nnodes>1 because we only use 1 task per node.